sys/net/gnrc/pktdump/gnrc_pktdump.c

```diff
@@ -201,6 +207,11 @@ kernel_pid_t gnrc_pktdump_init(void)
         gnrc_pktdump_pid = thread_create(_stack, sizeof(_stack), GNRC_PKTDUMP_PRIO,
                              0,
                              _eventloop, NULL, "pktdump");
+
+        gnrc_netreg_entry_t dump = GNRC_NETREG_ENTRY_INIT_PID(GNRC_NETREG_DEMUX_CTX_ALL,
+                                                                     gnrc_pktdump_pid);
+        _netreg_dump = dump;
+        gnrc_netreg_register(CONFIG_GNRC_PKTBUF_NETTYPE, &_netreg_dump);
```

Comment:
That seems unrelated

Reply:
Is it worth to be split out or is there a reason not to have this in `gnrc_pkdump.c`

Reply:
It's fine to keep it if it makes your life easier, but from the commit message alone I don't understand what it does and how it relates to this PR.